_assets/ci/Jenkinsfile

```diff
@@ -59,12 +67,20 @@ node('linux') {
         def uploadSpec = """{
             "files": [
                 {
-                    "pattern": "build/bin/statusgo-android-16.aar",
+                    "pattern": "build/lib/statusgo-android-16.aar",
                     "target": "libs-release-local/status-im/status-go/${version}/status-go-${version}.aar"
                 },
                 {
-                    "pattern": "build/bin/statusgo-ios-9.3-framework/status-go-ios.zip",
+                    "pattern": "build/lib/statusgo-ios-9.3-framework/status-go-ios.zip",
                     "target": "libs-release-local/status-im/status-go-ios-simulator/${version}/status-go-ios-simulator-${version}.zip"
+                },
+                {
+                    "pattern": "build/lib/statusgo-darwin-10.6-amd64.*",
+                    "target": "libs-release-local/status-im/status-go-darwin/${version}/status-go-darwin-${version}.zip"
```

Comment:
hmm, aren't we getting rid of Artifactory?

Reply:
good question, I didn't think of mixing that in this PR, but it would probably be useful to get @jakubgs view here. What should we do here instead?

Reply:
let's ask @jakubgs 
he used Digial Ocean Spaces for APK uploads at least.

Reply:
Yes, we are getting rid of Artifactory, and yes, we do use Digital Ocean spaces:
https://github.com/status-im/status-react/blob/develop/ci/common.groovy#L94
Like so:
https://github.com/status-im/status-react/blob/develop/ci/Jenkinsfile.combined#L63
I'm using `findFiles()` glob because I don't know the timestamps. Maybe you won't have to.
If you need additional space created for your artifacts just ping me.

(though I'm OoO today, i can do it later today)

Reply:
Awesome, I'll do that then. Thanks for the input!

Reply:
OK, new version now uploads everything to the same place as status-react. @jakubgs let me know if you have any preferences on where the artifacts should be going to.

Reply:
I've been working on the matching PR on the react side, and I think for simplicity sake, we should split this work in two: first generate the Desktop libs and ensure that works correctly, and then get rid of Artifactory. We can leave the code that uploads to DO spaces here, but also upload it to Artifactory until a future PR gets rid of it.